main.go

```diff
@@ -31,6 +31,9 @@ func main() {
 		case "compute-event":
 			actions.ComputeEvent(a)
 			break
+		case "compute-event-chart":

```

Comment:
perfect. in the future we have an interface called an action-runner in the cc-go-sdk impleementing that interface reduces the need to update a central switch case, and is generally the direction we are headed.